sql/core/src/test/scala/org/apache/spark/sql/collation/CollationTypePrecedenceSuite.scala

```diff
@@ -438,6 +438,32 @@ class CollationTypePrecedenceSuite extends QueryTest with SharedSparkSession {
       sql(s"SELECT map('key1', 'val1' collate utf8_lcase, 'key2', 'val2' collate unicode)"))
   }
 
+  test("user defined cast on maps") {
+    checkAnswer(
+      sql(s"""
+             |SELECT map_contains_key(
+             |  map('a' collate utf8_lcase, 'b'),
+             |  'A' collate utf8_lcase)
+             |""".stripMargin),
```

Comment:
Should 2 spaces indentation to `sql` (see https://github.com/databricks/scala-style-guide?tab=readme-ov-file#spacing-and-indentation). Please, fix here and below.
```suggestion
      sql(s"""
        |SELECT map_contains_key(
        |  map('a' collate utf8_lcase, 'b'),
        |  'A' collate utf8_lcase)""".stripMargin),
```

Reply:
done!